### PR TITLE
Include manifest or class in missing serializer

### LINKED ISSUE #26217: Log the manifest when serializer was not found
When during deserialization we encounter a message with a serializer id that we no longer recognize, we produce a 'Cannot find serializer with id [...]' message. To make it easier to analyze what is going on, it could be useful to also log the manifest that was sent along with the message, if any.

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #26217

### CODE REVIEW

File: akka-actor/src/main/scala/akka/serialization/Serialization.scala [hunk 1]
```diff
@@ -188,8 +188,9 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     Try {
       val serializer = try getSerializerById(serializerId) catch {
         case _: NoSuchElementException ⇒ throw new NotSerializableException(
-          s"Cannot find serializer with id [$serializerId]. The most probable reason is that the configuration entry " +
-            "akka.actor.serializers is not in synch between the two systems.")
+          s"Cannot find serializer with id [$serializerId] ${clazz.map(c ⇒ "(class [" + c.getName + "]").getOrElse()}. " +
```
Comment (raboof) [inline akka-actor/src/main/scala/akka/serialization/Serialization.scala]:
```suggestion
          s"Cannot find serializer with id [$serializerId]${clazz.map(c ⇒ " (class [" + c.getName + "])").getOrElse("")}. " +
```
I think `None.getOrElse().toString` would be `"()"` where I guess we'd want `""`?

**Review by raboof — COMMENTED** [review]
Should we also change the instance around line 257?

File: akka-actor/src/main/scala/akka/serialization/Serialization.scala (hunk 1 above)
Comment (johanandren) [reply akka-actor/src/main/scala/akka/serialization/Serialization.scala]: ah, ups should be ""

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
Unrelated failure (#26223)

**Comment by johanandren** [comment]
PLS BUILD

**Comment by raboof** [comment]
PLS BUILD

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**